src/data/builder-tools.js

```diff
@@ -161,6 +161,12 @@ const Showcases = [
     website: "https://cardano-community.github.io/guild-operators/",
     getstarted: "/docs/operate-a-stake-pool/guild-ops-suite",
     tags: ["featured", "getstarted", "operatortool"],
+    title: "CNTools",
```

Comment:
@Mercurial - You might have accidentally used previous commit as base

Reply:
oh I merged it wrong should be a quick fix